server/models/Column.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require("mongoose");
+
+const columnSchema = new mongoose.Schema({
+  _id: {
+    type: mongoose.Types.ObjectId,
+    required: true,
+  },
```

Comment:
Ids are automatically added onto by Mongoose. We don't need to explicitly set this.